docs/resources/cce_cluster.md

```diff
@@ -134,7 +134,7 @@ versions are available, choose Dashboard > Buy Cluster on the CCE console. Chang
 * `authenticating_proxy_ca` - (Optional, String, ForceNew) CA root certificate provided in the authenticating_proxy mode. The CA root certificate
 	is encoded to the Base64 format. Changing this parameter will create a new cluster resource.
 
-* `multi_az` - (Optional, Bool, ForceNew) Enable multiple AZs for the cluster, only when using HA flavors. Changing this parameter will create a new cluster resource.
+* `multi_az` - (Optional, Bool, ForceNew) Enable multiple AZs for the cluster, only when using HA flavors. Changing this parameter will create a new cluster resource. Conflicts with `masters`.
```

Comment:
two small tips:
1. if the paramter conflicts with others, please use the following template:
*This parameter and `xxx` are alternative.*
2. In code style guides, maximum line length rules are often set to 100 or 120.

Reply:
fixed